blockchain/types/src/block.rs

```diff
@@ -384,30 +384,24 @@ mod block_tests {
     /// we add/change Block/BlockContents and accidentally break id
     /// calculation of old blocks.
     fn test_hashing_is_consistent_block_version_one() {
-        // FIXME (#2042): This shouuld not use StdRng, because the rust standard library
-        // sometimes updates what RNG algorithm this point to. We should
-        // rerun this with Hc128 RNG (from mc_util_test_helper) and update the
-        // test vectors.
-        use rand::rngs::StdRng;
-
-        let mut rng = StdRng::from_seed([1u8; 32]);
+        let mut rng = get_seeded_rng();
```

Comment:
Nit: https://doc.rust-lang.org/1.0.0/style/style/imports.html#prefer-fully-importing-types/traits-while-module-qualifying-functions.

Reply:
Per that style-guide, importing the function without module-qualification is appropriate as a convenience in test sub-modules